apps/prairielearn/src/components/QuestionContainer.tsx

```diff
@@ -383,11 +388,11 @@ export function QuestionTitle({
 }: {
   questionContext: QuestionContext;
   question: Question;
-  questionNumber: string;
+  questionNumber?: string;
 }) {
-  if (questionContext === 'student_homework') {
+  if (questionNumber != null && questionContext === 'student_homework') {
```

Comment:
Nit: is it worth treating this as an invariant instead and doing a runtime assertion?